libraries/AP_ADSB/AP_ADSB.h

```diff
@@ -23,26 +23,40 @@
 
 #include <AP_HAL/AP_HAL.h>
 #include <AP_HAL/AP_HAL_Boards.h>
-#include <AP_Common/AP_Common.h>
-#include <AP_Param/AP_Param.h>
-#include <AP_Common/Location.h>
-#include <GCS_MAVLink/GCS_MAVLink.h>
 
 #ifndef HAL_ADSB_ENABLED
 #define HAL_ADSB_ENABLED !HAL_MINIMIZE_FEATURES && BOARD_FLASH_SIZE > 1024
 #endif
 
 #if HAL_ADSB_ENABLED
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Common/Location.h>
+#include <GCS_MAVLink/GCS_MAVLink.h>
+
+#define ADSB_BITBASK_RF_CAPABILITIES_UAT_IN         (1 << 0)
```

Comment:
they were moved to the header when Sagetech was using them, but I have since removed that. I've moved them back to the cpp file in https://github.com/ArduPilot/ardupilot/pull/15610